uaclient/cli/__init__.py

```diff
@@ -690,6 +690,9 @@ def action_config_set(args, *, cfg, **kwargs):
         raise exceptions.InvalidArgChoice(
             arg="<key>", choices=", ".join(config.UA_CONFIGURABLE_KEYS)
         )
+    if not set_value.strip():
+        subparser.print_help()
```

Comment:
Do we need to print the help message here ? Maybe just stating that the user provided an invalid empty value is enough.

What do you think @renanrodrigo ?

Reply:
This is following what we do for all other errors on setting config values. I think this looks good as is.